src/css/_Styles.scss

```diff
@@ -97,35 +163,90 @@ body {
   margin: 0;
 }
 
-//.trip-destination {
-//  color: #0f5479;
-//  font-size: 35px;
-//}
-
 .card-text {
   display: flex;
   flex-direction: column;
 }
 
+p{
+  color: $lightYellow;
+  font-size: 1.0em;
+  background-color: #5e7c91;
+  border-radius: 2px;
+}
 
-@media screen and (min-width: 992px) {
-  .login-view {
-    margin: 5% 10%;
+.hidden {
+  display: none;
+}
+
+@media screen and (max-width: 800px) {
+  .input-options {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    width: 50vw;
+    justify-self: center;
   }
-  .login-out-button {
-    height: 2.6vw;
-    width: 5.6vw;
+
+  select {
+    width: 140px;
   }
-}
 
-@media screen and (min-width: 600px) {
-  .login-view {
-    margin: 3% 15%;
+
+    .number-travelers {
+      grid-area: travelers;
+    }
+
+    .possible-destination {
+      grid-area: destination;
+    }
   }
 
-}
-@media screen and (max-width: 600px) {
-  .login-view {
-    margin: 5% 8%;
+  @media screen and (min-width: 992px) {
+    .login-view {
+      margin: 5% 10%;
+    }
+    .login-out-button {
+      height: 2.6vw;
+      width: 5.6vw;
+    }
   }
-}
+
+  @media screen and (min-width: 600px) {
```

Comment:
It might be helpful to put these media queries into a partial file & import them at the bottom of your `base.scss` file